setup.py

```diff
@@ -10,60 +10,56 @@
 with open('kiwipy/version.py') as f:
     exec(f.read(), about)
 
-setup(
-    name="kiwipy",
-    version=about['__version__'],
-    description='A python remote communications library',
-    long_description=open('README.rst').read(),
-    url='https://github.com/muhrin/kiwipy.git',
-    author='Martin Uhrin',
-    author_email='martin.uhrin@gmail.com',
-    license=__license__,
-    classifiers=[
-        'Development Status :: 4 - Beta',
-        'License :: OSI Approved :: MIT License',
-        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-    ],
-    keywords='communication messaging rpc broadcast',
-    install_requires=[
-        'six',
-        'shortuuid',
-        'typing; python_version<"3.5"',
-        'enum34; python_version<"3.4"',
-        'backports.tempfile; python_version<"3.2"',
-        'futures; python_version == "2.7"',
-    ],
-    extras_require={
-        'rmq': [
-            'pika>=1.0.0b2',
-            'topika>=0.2.0, <0.3.0',
-            'tornado<6; python_version<"3"',
-            'tornado<5; python_version>="3"',
-            'pyyaml'
-        ],
-        'dev': [
-            'pip',
-            'pre-commit',
-            'pytest>=4',
-            'pytest-cov',
-            'ipython<6',
-            'twine',
-            'yapf',
-            'prospector',
-            'pylint<2; python_version<"3"',
-            'pylint; python_version>="3"',
-        ],
-        "docs": [
-          "Sphinx==1.8.4",
-          "Pygments==2.3.1",
-          "docutils==0.14",
-        ],
-    },
-    packages=['kiwipy', 'kiwipy.rmq'],
-    test_suite='test')
+setup(name="kiwipy",
+      version=about['__version__'],
+      description='A python remote communications library',
+      long_description=open('README.rst').read(),
+      url='https://github.com/muhrin/kiwipy.git',
+      author='Martin Uhrin',
+      author_email='martin.uhrin@gmail.com',
+      license=__license__,
+      classifiers=[
+          'Development Status :: 4 - Beta',
+          'License :: OSI Approved :: MIT License',
+          'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+          'Programming Language :: Python :: 2',
+          'Programming Language :: Python :: 2.7',
+          'Programming Language :: Python :: 3.4',
+          'Programming Language :: Python :: 3.5',
+          'Programming Language :: Python :: 3.6',
+          'Programming Language :: Python :: 3.7',
+      ],
+      keywords='communication messaging rpc broadcast',
+      install_requires=[
```

Comment:
While we're at it, do you want to specify ``python_requires`` here? This allows ``pip`` to install the correct version once support for a specific Python version is dropped (but needs to be in a release _before_ it is dropped).

See e.g.: https://github.com/aiidateam/aiida-core/pull/3574

Reply:
Done